tests/plugin/common.py

```diff
@@ -1,40 +1,42 @@
 """Common methods for plugin tests"""
 
+from typing import Optional
+
 import allure
 
 
 def run_tests(
     testdir,
-    filename: str = None,
-    py_file: str = None,
-    additional_opts: list = None,
+    testfile_path: str = "",
+    makepyfile_str: str = "",
+    additional_opts: Optional[list] = None,
     outcomes=None,
 ):
     """
     Run tests with pytest parameters from .py file or multiline string
     :param testdir:
-    :param filename: name of file from the directory "_test_files" which will be running
-    :param py_file: multiline string for makepyfile method which will be running if param 'filename' is None
+    :param testfile_path: name of file from the directory "_test_files" which will be running
+    :param makepyfile_str: multiline string for makepyfile method which will be running if param 'testfile_path' is None
     :param additional_opts: list of additional pytest launch parameters
     :param outcomes: optional outcomes expect. Ex. {"failed":1}
     """
-    if additional_opts is None:
-        additional_opts = []
-    if filename:
-        testdir.copy_example(filename)
-    elif py_file:
-        testdir.makepyfile(py_file)
+    assert testfile_path or makepyfile_str, "At least one of the `testfile_path` or `makepyfile_str` should be passed."
```

Comment:
It's better to `raise ValueError()` here. This is a test run error, not an assertion error in tests

Reply:
Done.